src/Sql/Sql/help/New-AzSqlInstance.md

```diff
@@ -242,7 +379,7 @@ Parameter Sets: (All)
 Aliases: InstanceName
 
 Required: True
-Position: 0
+Position: 1
```

Comment:
@johnpaulkee same comment about reversing the positions of `-Name` and `-ResourceGroupName`

Reply:
Acked.